marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobDocManager.java

```diff
@@ -46,7 +46,7 @@ public JsonNode postJobs(String jobId, String status, String step, String lastCo
         try {
             resultItr = this.getServices().post(params, new StringHandle("{}").withFormat(Format.JSON));
         } catch (Exception e) {
-            throw new RuntimeException("Unable to update the job document");
+            throw new RuntimeException("Unable to update the job document; cause: " + e.getMessage(), e);
```

Comment:
This isn't related to running flows, but I ran into this a few times while testing out flow running, and the lack of an error message or the original exception made debugging more difficult, so figured I'd improve that.